src/main/java/io/fabric8/maven/docker/access/UrlBuilder.java

```diff
@@ -65,6 +65,11 @@ public String deleteImage(String name, boolean force) {
                 .build();
     }
 
+    public String getImage(ImageName name) {
+        return u("images/%s/get", name.getNameWithoutTag())
```

Comment:
I don't think we should use the name without tag because according to the Docker reference manual in that case all image of all tags are returned. Actually we should use the full image name here.